chip/Cargo.lock

```diff
@@ -0,0 +1,7 @@
+# This file is automatically @generated by Cargo.
```

Comment:
Cargo.lock nicht einchecken

Reply:
dachte du haettest mal was gesagt, dass man sie einchecken soll 
mache sie raus

Reply:
ne, nur die tomls. Die Locks erzeugt dir Cargo, wenn Du das erste mal den Branch baust

Reply:
ich hab in meinem Branch gerade alle Lockfiles auf gitignore gesetzt. kann man bei dir also lassen, die verschwinden dann demnächst von selbst.